nitransforms/nonlinear.py

```diff
@@ -22,9 +31,24 @@ class DisplacementsFieldTransform(TransformBase):
     __slots__ = ["_field"]
 
     def __init__(self, field, reference=None):
-        """Create a dense deformation field transform."""
+        """
+        Create a dense deformation field transform.
+
+        Example
+        -------
+        >>> DisplacementsFieldTransform(test_dir / "someones_displacement_field.nii.gz")
+        <(57, 67, 56) field of 3D displacements>
```

Comment:
Generally expect class name to come first.

```suggestion
        <DisplacementFieldTransform[3D] (57, 67, 56)>
```